livestyled/models/order.py

```diff
@@ -157,3 +157,13 @@ def placeholder(cls, id):
             external_id=None,
             seat_info=None
         )
+
+    def diff(self, other):
+        differences = {}
+        fields = (
```

Comment:
can fit in one line. What of the other properties. We should just solve this problem at once so we don't have to come back to add it later.